src/main/resources/org/broadinstitute/hellbender/tools/walkers/vqsr/nvscorevariants.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python3
+
+import argparse
+import torch
+import sys
+import pytorch_lightning as pl
+from lightning_fabric.utilities import seed
+from torch.utils.data import DataLoader
+from scorevariants.dataset import ReferenceDataset
+from scorevariants.readers import TensorReader, ReferenceTensorReader
+from scorevariants.models.wrapper import LightningWrapper
+from scorevariants.create_output_vcf import create_output_vcf
+
+def get_model(args, model_file):
+    """
+    Args:
+        args: model specific and trainer arguments
+        model_file: model file to load parameters from a pretrained PyTorch model
+    """
+    dict_args = vars(args)
+    model = torch.load(model_file)
+    model = LightningWrapper(model, **dict_args)
+    return model
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--vcf-file', required=True, help='A VCF file containing variants to creat datasets')
+    parser.add_argument('--ref-file', required=True, help='Reference sequence file to creat datasets')
+    parser.add_argument('--input-file', help='BAM file containing reads to creat datasets')
+    parser.add_argument('--tensor-type', default='reference', help='Name of the tensors to generate, reference for 1D reference tensors and read_tensor for 2D tensors.')
+    parser.add_argument('--batch-size', type=int, default=32, help='Batch size')
+    parser.add_argument('--seed', type=int, default=724, help='Seed to initialize the random number generator')
+    parser.add_argument('--tmp-file', default='tmp.txt', help='The temporary VCF-like file where variants scores will be written')
+    parser.add_argument('--output-file', required=True, help='Output VCF file')
+    parser.add_argument('--gpus', type=int, nargs='+', help='Number of GPUs (int) or which GPUs (list)')
```

Comment:
Okay, so I figured this one out.  The `gpus` arg was deprecated in lightning 1.7 and removed in 2.0.  Best I can tell from reading the lightning docs, the way this information is now meant to be supplied to the `Trainer` object is using two arguments: `accelerator` for picking the type of processor (cpu, gpu, etc.), and `devices` for specifying how many to use.

Reply:
Probably worth wiring this up, then!

Reply:
Ah, makes sense.  Should we expose that? If you don't specify but you have a GPU available what does it do by default?  We don't want it to ignore available compute.